### PR TITLE
LPS-187448 | Poshi Automation for Bug | Master

### PR DESCRIPTION
**Pull request by rafaellajordao**
# Poshi Automation for Bug LPS-143093 Item types configuration for a collection

Hi! :)
This pr is corresponding to this ticket: [LPS-143093](https://liferay.atlassian.net/browse/LPS-143093) → [LPS-187448](https://liferay.atlassian.net/browse/LPS-187448)

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/collections/Collections.testcase [hunk 1]
```diff
@@ -146,6 +146,44 @@ definition {
...
 
+	@description = "This is a test for LPS-143093. Item types configuration for a collection"
+	@priority = 4
+	test CheckItemTypesForCollection {
+		task ("Given the option of Select Types, under Multiple Item Types") {
+			AssetListsAdmin.openAssetListsAdmin(siteURLKey = "test-site-name");
+
+			AssetListsAdmin.addManualSelection(assetListTitle = "Manual Asset List Test");
+
+			Select(
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/collections/Collections.testcase:157]: AssetListsAdmin.configureItemTypeInManualCollection

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/collections/Collections.testcase [hunk 2]
```diff
@@ -146,6 +146,44 @@ definition {
...
+				value1 = "Select Types...");
+		}
+
+		task ("When selecting some types for the Available box") {
+			for (var key_type : list "Blogs Entry,Web Content Article") {
+				Click.clickNoMouseOver(locator1 = "AssetLists#ASSET_LIST_ENTRY_TYPE_SELECTED");
+
+				Click(
+					key_title = "Move selected items from Selected to Available.",
+					locator1 = "Button#ANY_TITLE");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/collections/Collections.testcase:168]:
```
				Click(locator1 = "APConfiguration#SELECT_MORE_THAN_ONE_ASSET_TYPE_MOVE_AVAILABLE");

```

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/collections/Collections.testcase [hunk 3]
```diff
@@ -146,6 +146,44 @@ definition {
...
+			}
+
+			PortletEntry.save();
+		}
+
+		task ("Then only selected types are shown in the selection button.") {
+			Button.clickSelect();
+
+			for (var menuItem : list "Account Entry,Bookmarks Entry,Bookmarks Folder,Calendar Event,Basic Document,Documents Folder,Dynamic Data Lists Record,Knowledge Base Article,Message Boards Message,Translation,Web Content Folder,Wiki Page") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/collections/Collections.testcase:177]: For case stability, No need to assert all
Account Entry, Calendar Event,Basic Document,Documents Folder,Knowledge Base Article,Message Boards Message,Translation,Web Content Folder, Wiki Page

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/collections/Collections.testcase [hunk 4]
```diff
@@ -146,6 +146,44 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-143093. Item types configuration for a collection"
+	@priority = 4
+	test CheckItemTypesForCollection {
```
[the remark this answers is not in the document]
Comment (rafaellajordao) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/collections/Collections.testcase:151]: done!

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/collections/Collections.testcase (hunk 2 above)
Comment (rafaellajordao) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/collections/Collections.testcase:168]: done!

**Comment by Tim-Cao** [comment]
@rafaellajordao  See changes
https://github.com/liferay-echo/liferay-portal/pull/14233

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**